android/app/build.gradle

```diff
@@ -0,0 +1,66 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "org.freedesktop.gstreamer.play"
+        minSdkVersion 19
+        targetSdkVersion 22
```

Comment:
You are right, it is too high.
I suggest to use 16 for the App, mostly because I do not have any older devices. ;-)
Anything lower will catch only about  < 10% of the devices in the market (according to https://developer.android.com/about/dashboards/index.html).